sdk/keyvault/keyvault-secrets/review/keyvault-secrets.review.api.md

```diff
@@ -0,0 +1,151 @@
+## API Report File for "@azure/keyvault-secrets"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { HttpClient } from '@azure/core-http';
+import { HttpPipelineLogger } from '@azure/core-http';
+import * as msRest from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { RequestOptionsBase } from '@azure/core-http';
+import { ServiceClientOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface DeletedSecret extends Secret {
+    readonly deletedDate?: Date;
+    recoveryId?: string;
+    readonly scheduledPurgeDate?: Date;
+}
+
+// @public
+export type DeletionRecoveryLevel = "Purgeable" | "Recoverable+Purgeable" | "Recoverable" | "Recoverable+ProtectedSubscription";
+
+// @public
+export interface GetSecretOptions {
+    requestOptions?: msRest.RequestOptionsBase;
+    version?: string;
+}
+
+// @public
+export interface GetSecretsOptions {
+    requestOptions?: msRest.RequestOptionsBase;
+}
+
+// @public
+export interface NewPipelineOptions {
+    // (undocumented)
+    HTTPClient?: HttpClient;
+    // (undocumented)
+    logger?: HttpPipelineLogger;
+    // (undocumented)
+    proxyOptions?: ProxyOptions;
+    // (undocumented)
+    retryOptions?: RetryOptions;
+    telemetry?: TelemetryOptions;
+}
+
+export { PagedAsyncIterableIterator }
+
+export { PageSettings }
+
+// @public (undocumented)
+export interface ParsedKeyVaultEntityIdentifier {
+    name: string;
+    vaultUrl: string;
+    version?: string;
+}
+
+// @public
+export interface ProxyOptions {
+    // (undocumented)
+    proxySettings?: string;
+}
+
+// @public
+export interface RetryOptions {
+    readonly maxRetryDelayInMs?: number;
+    readonly retryCount?: number;
+    readonly retryIntervalInMS?: number;
+}
+
+// @public
+export interface Secret {
+    properties: SecretProperties;
+    value?: string;
+}
+
+// @public
+export interface SecretProperties extends ParsedKeyVaultEntityIdentifier {
+    contentType?: string;
+    readonly created?: Date;
+    enabled?: boolean;
+    expires?: Date;
+    id?: string;
+    readonly keyId?: string;
+    readonly managed?: boolean;
+    notBefore?: Date;
+    readonly recoveryLevel?: DeletionRecoveryLevel;
+    tags?: {
+        [propertyName: string]: string;
+    };
+    readonly updated?: Date;
+}
+
+// @public
+export class SecretsClient {
```

Comment:
`SecretClient`?

Reply:
I think I'm vaguely +1 on the singular name for the clients as well (so, KeyClient as well).